docs/OV_Runtime_UG/preprocessing_overview.md

```diff
@@ -81,16 +81,16 @@ To address particular input of model/preprocessor, use `ov::preprocess::PrePostP
 
 @endsphinxtabset
 
-Here we've specified all information about user's input:
- - Precision is U8 (unsigned 8-bit integer)
- - Data represents tensor with {1,480,640,3} shape
- - [Layout](./layout_overview.md) is "NHWC". It means that 'height=480, width=640, channels=3'
- - Color format is `BGR`
+Below is all the specified input information:
```

Comment:
```suggestion
Below are all the specified input information:
```

Reply:
> 

```suggestion
Below is all the specified input information:
```

The "information" noun is an uncountable, therefore it does not take plural.